Adding metriq qedc blog post.

vprusso opened the pull request:
Adding blog post for Metriq <-> QED-C pipeline.

nathanshammah commented on posts/_markdown/metriq_qedc.md:
```diff
@@ -0,0 +1,57 @@
+measure the effectiveness of quantum computing hardware for specific
+applications (further information provided in this
+[preprint](https://arxiv.org/abs/2110.03137)). This provided some of the first
+application-oriented benchmarks that we made public with supporting data along
```
```suggestion
application-oriented benchmarks that were made public with supporting data along
```

nathanshammah reviewed (changes requested):
Looks great, thanks for the nice additions @vprusso regarding the Python client. You just need to update the index at blog.html and generate the html file from this md with the instructions from the Readme.

willzeng commented on posts/_markdown/metriq_qedc.md:
```diff
@@ -0,0 +1,57 @@
+
+We are excited to announce the integration of an automated pipeline to add
+state-of-the-art benchmark from the Quantum Economic Development Consortium
+([QED-C](https://quantumconsortium.org/)) into **Metriq**, the open platform that
```
Should be a hyperlink to metriq!

willzeng commented:
Made one small comment but this looks great to ship from me!

vprusso commented:
> d91beb9

Covered in [d91beb9](https://github.com/unitaryfund/unitary-fund/pull/300/commits/d91beb9e7e3502f1c5404cecde93a1e1c607b32f)

vprusso commented:
> Made one small comment but this looks great to ship from me!

Covered in [d91beb9](https://github.com/unitaryfund/unitary-fund/pull/300/commits/d91beb9e7e3502f1c5404cecde93a1e1c607b32f)

vprusso commented:
Addressed comments from @natestemen in [d91beb9](https://github.com/unitaryfund/unitary-fund/pull/300/commits/d91beb9e7e3502f1c5404cecde93a1e1c607b32f)

nathanshammah commented:
@vprusso one small thing we missed: Can you please change the files names to 2023_metriq_qedc as it's our current format? Thanks.

WrathfulSpatula commented on posts/metriq_qedc.html:
```diff
@@ -0,0 +1,168 @@
+
+                            <div>
+                                <p>We are excited to announce the integration of an automated pipeline to add state-of-the-art benchmark from the Quantum Economic Development Consortium (<a href="https://quantumconsortium.org/">QED-C</a>) into <strong>Metriq</strong>, the open platform that makes transparent, accessible benchmarks available to everyone in the quantum computing community.</p>
+                                <p>The Technical Advisory Committee (TAC) for Standards and Performance Metrics from QED-C provided an open-source suite of quantum benchmarks (<a href="https://github.com/SRI-International/QC-App-Oriented-Benchmarks">GitHub</a>) to measure the effectiveness of quantum computing hardware for specific applications (further information provided in this <a href="https ...
```
Same comment as above: m dash (—)

WrathfulSpatula commented on posts/_markdown/metriq_qedc.md:
```diff
@@ -0,0 +1,57 @@
+applications (further information provided in this
+[preprint](https://arxiv.org/abs/2110.03137)). This provided some of the first
+application-oriented benchmarks that were made public with supporting data along
+with a preprint–an incredibly valuable asset to the benchmarking community.
```
Alternatively, just make it a comma to indicate a parenthetical phrase.

vprusso replied:
I loved this comment--great attention to detail! Fixed in [568509d](https://github.com/unitaryfund/unitary-fund/pull/300/commits/568509d2de2f168229634c6bd0eff87385a67a23)

vprusso commented:
Updated and responded to all comments: [568509d](https://github.com/unitaryfund/unitary-fund/pull/300/commits/568509d2de2f168229634c6bd0eff87385a67a23)

nathanshammah commented on blog.html:
```diff
@@ -72,6 +72,10 @@
               <section class="light">
                 <ul style="list-style-type: none">
+                  <li class="leading-block">
+                    <span class="blog-post-date">23 February, 2023</span>
```
```suggestion
                    <span class="blog-post-date">28 February, 2023</span>
```

nathanshammah commented on posts/2023_metriq_qedc.html:
```diff
@@ -0,0 +1,168 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8">
+    <meta name="date" content='2023-02-23'>
```
```suggestion
    <meta name="date" content='2023-02-28'>
```

nathanshammah commented on posts/2023_metriq_qedc.html:
```diff
@@ -0,0 +1,168 @@
+                                <h2 class="leading-arrow">Automatically running QED-C benchmarks on Metriq</h2>
+                                <h3></h3>
+                            </right>
+                            2023-02-23
```
```suggestion
                            2023-02-28
```

vprusso commented:
Awesome. Thank you for the updates, @nathanshammah !

vprusso commented:
MWR! 🚀
